src/main/java/duke.txt

```diff
@@ -0,0 +1,7 @@
+[E][X] project (at: Sep 9 2019)
```

Comment:
As mentioned by cowlinn, perhaps consider including your output file in the `.gitignore`? This will prevent unnecessary changes to your commits